### PR TITLE
fix: alignment of member profile icon and copy

### PR DESCRIPTION
**Pull request by yeganathan18**
# Changes proposed ✍️

### <samp>🤖 Generated by Copilot at 8f82834</samp>

Adjusted the text size and margin of the sentiment icons and labels in the activity timeline component. This improves the UI consistency and readability of the activity messages.
​

### <samp>🤖 Generated by Copilot at 8f82834</samp>

> _`text-[14px]` now_
> _icons and messages fit_
> _autumn of clutter_

### <samp>🤖 Generated by Copilot at 8f82834</samp>

*  Adjust the text size and margin of the sentiment icons and labels in the activity timeline ([link](https://github.com/CrowdDotDev/crowd.dev/pull/822/files?diff=unified&w=0#diff-c67285c5cd2a33322ce7215db3b710ffc55f5b50b612c7748b953675bbb45312L10-R18), [link](https://github.com/CrowdDotDev/crowd.dev/pull/822/files?diff=unified&w=0#diff-ac35928641997d2be2d0c43afd02a85bf32a2b169074ac40ddcad8b21477eaf8L76-R76))

## Checklist ✅
- [ ] Label appropriately with `Feature`, `Improvement`, or `Bug`.
- [x] Add screehshots to the PR description for relevant FE changes
- [ ] New backend functionality has been unit-tested.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [x] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.

### CODE REVIEW

**Comment by yeganathan18** [comment]
Fixes #627 

here's the screenshot,

image

**Review by joanagmaia — CHANGES_REQUESTED** [review]
All looking great 😄 
Thanks for another contribution 💪 

Just left one small request for change. After that should be ready to go

File: frontend/src/modules/activity/components/activity-sentiment.vue [hunk 1]
```diff
@@ -7,15 +7,15 @@
   >
     <i
       v-if="label === 'Positive'"
-      class="ri-emotion-happy-line text-green-600 text-base"
+      class="ri-emotion-happy-line text-green-600 text-[14px]"
```
[the remark this answers is not in the document]
Comment (yeganathan18) [reply frontend/src/modules/activity/components/activity-sentiment.vue:10]: Oops! Sorry for that, I didn't check the `tailwind.config.js` 😬

[the remark this answers is not in the document]
Comment (yeganathan18) [reply frontend/src/modules/activity/components/activity-sentiment.vue:10]: Updated the PR with changes, and thanks for letting me know!

**Review by joanagmaia — APPROVED** [review]
LGTM 🚀

**Pull request merged**